frontend/views/containers/group-settings/GroupVotingSystem.vue

```diff
@@ -0,0 +1,133 @@
+<template lang='pug'>
+  ul.c-wrapper
+    li.cardBox(
+      v-for='option in votingSytemSorted'
+      :class='{ isActive: isVotingActive(option) }'
+      :aria-current='isVotingActive(option) ? "true" : "false"'
```

Comment:
What's this `aria-current` thing and why is it `"true"` instead of `true`?

Can you add a comment explaining this above line 3? (Or a link to an explainer?)

Reply:
Here's [an article about aria-current](https://tink.uk/using-the-aria-current-attribute/). It's used to mark an element in a list as the "current/active" one. The same way we show it visually (with the ACTIVE pill), we should also say it programmatically.

Regarding `true/"true"` in this case does not make a difference.
Code simplified to `:aria-current='isVotingActive(option)'`.